TWLight/templates/about.html

```diff
@@ -165,7 +165,7 @@ <h3>{% trans "EZProxy" %}</h3>
     {% endblocktrans %}
   </p>
 
-  <h3>Library Bundle</h3>
+  <h3 id="anchor">Library Bundle</h3>
```

Comment:
The name of the id does not indicate what it's identifying. I suggest changing the id name to `bundle-section`